src/lib/Scenes/Fair/Components/FairBoothHeader.tsx

```diff
@@ -28,16 +29,43 @@ const ButtonWrapper = styled(Box)`
   height: 85;
 `
 
+const track: Track<Props, State> = _track
+
+@track()
 export class FairBoothHeader extends React.Component<Props, State> {
   state = {
     isFollowedChanging: false,
   }
 
+  @track((props, _, args) => {
+    const partnerSlug = args[0]
```

Comment:
I think this could fail for fairs that have partner === `null` but in this case we shouldn't show the follow button at all.

Reply:
Are you sure? This function is only called when a partner is followed or unfollowed so I dont think the partner can be `null`